core/src/main/java/io/confluent/connect/storage/tools/SchemaSourceTask.java

```diff
@@ -86,6 +91,54 @@
       .field("seqno", Schema.INT64_SCHEMA)
       .build();
 
+  private static Schema simpleKeySchema = Schema.STRING_SCHEMA;
+  private static Schema complexKeySchema = SchemaBuilder.struct().name("complexkeyschema")
```

Comment:
```suggestion
  private static Schema complexKeySchema = SchemaBuilder.struct().name("complex-key-schema")
```

Reply:
This is a tricky one, I had to remove the dashes because tests were failing; it looks like field names in avro schemas don't support special characters. 
` ERROR Could not parse Avro schema (io.confluent.kafka.schemaregistry.avro.AvroSchemaProvider:43)
org.apache.avro.SchemaParseException: Illegal character in: complex-key-schema` https://avro.apache.org/docs/1.7.7/spec.html#Names